charts/ctlog/templates/secret-operator-rolebinding.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if not .Values.server.config.secret.enabled }}
```

Comment:
should we do this for secret-operator-role.yaml as well?

Reply:
yup sg, also added it to `ctlog-configmap.yaml`